WordPress/Classes/Services/PostService.h

```diff
@@ -1,128 +1,7 @@
-#import <Foundation/Foundation.h>
-#import "LocalCoreDataService.h"
-#import "PostServiceOptions.h"
 
-@class Blog, Post, Page, AbstractPost;
-@class RemotePost;
-
-typedef NS_ENUM(NSInteger, PostServiceType) {
-    PostServiceTypePost,
-    PostServiceTypePage,
-    PostServiceTypeAny
-};
+typedef NSString * PostServiceType;
+extern NSString * const PostServiceTypePost;
+extern NSString * const PostServiceTypePage;
+extern NSString * const PostServiceTypeAny;
```

Comment:
Because swift can't make global variables visible to Objective-C, and ObjC can't see the swift String enum. There's a solution in Xcode 8 that allows Objective-C NSString typedefs to be compiled into strongly typed swift structs, but I couldn't find anything so elegant that works _now_. So I brought back the old PostServiceType (from a few revisions ago) and added an objective-C compatible method in an extension.

I'm open to alternate ideas on this.